backend/src/test/unimplemented/ID026_viewCollaborationResponses.feature

```diff
@@ -0,0 +1,85 @@
+    And the database contains the following CollaborationRequests:
+      | id | ideaId | userId | status   | additionalContact | message                                                                       |
+      | 23 | 17     | 2      | Pending  | "438-764-1940"    | "Hi, I would like to join this project!"                                      |
+      | 24 | 18     | 2      | Pending  | null              | "Hi, I have experience in that field and I am interested in working on this!" |
+      | 25 | 20     | 3      | Accepted | null              | "Hi, I have experience in that field and I am interested in working on this!" |
+      | 26 | 21     | 2      | Pending  | null              | "Hi, I have experience in that field and I am interested in working on this!" |
+      | 27 | 20     | 1      | Declined | null              | "Hi, I have experience in that field and I am interested in working on this!" |
+      | 28 | 17     | 3      | Accepted | null              | "Hi, I have experience in that field and I am interested in working on this!" |
+      | 29 | 18     | 3      | Declined | null              | "Hi, I have experience in that field and I am interested in working on this!" |
+      | 30 | 19     | 1      | Pending  | null              | "Hi, I have experience in that field and I am interested in working on this!" |
+    And the database contains the following CollaborationResponses:
+      | id | collaborationRequestId | additionalContact                 | message                                                          |
+      | 31 | 23                     | "Hi, my contact is +438-764-1940" | "Hi, could you develop on your experience in that field please." |
+      | 32 | 24                     | ""                                | ""                                                               |
+
+  # normal flow
+
+  Scenario Outline: successfully view collaboration responses for a collaboration request
+    Given I am logged in as the user with email "<email>" and password "<password>"
+    When the user requests to access the collaboration responses for the idea with id "<ideaId>"
+    Then the user shall have access to the collaboration response with id "<collaborationResponseIds>"
+
+    Examples:
+      | email              | password     | ideaId | collaborationResponseIds |
+      | jane.doe@gmail.com | P@ssWord1234 | 17     | 31                       |
+      | jane.doe@gmail.com | P@ssWord1234 | 28     | 32                       |
+
+  # alternate flow
+
+  Scenario Outline: try to access collaboration responses for an idea that does not have any responses yet
+    Given I am logged in as the user with email "<email>" and password "<password>"
+    When the user requests to access the collaboration responses for the idea with id "<ideaId>"
+    Then the user shall not have access to any collaboration responses
```

Comment:
How do we want to indicate this? I think lets go with returning an empty list, so could do an assertion on the length of the list that is returned